x-pack/plugins/triggers_actions_ui/public/application/sections/action_connector_form/action_type_form.tsx

```diff
@@ -250,6 +254,8 @@ export const ActionTypeForm = ({
 
   const showSelectActionGroup = actionGroups && selectedActionGroup && setActionGroupIdByIndex;
 
+  const showDefaultSummaryMessage = selectedNotifyWhen === RuleNotifyWhen.THROTTLE;
```

Comment:
I think it's a bit early to add this DefaultSummaryMessage.
`RuleNotifyWhen.THROTTLE` does not mean that the action is a summary action.
First we should add two more notifyWhen types to the options: `Summary of alerts per rule run` and `Summary of alerts on a custom interval`, then we can show the defaultSummaryMessage when either of them is selected.

Reply:
Okay sounds good! Will wait until those notifyWhen types are added

Reply:
Should be ready to be reviewed again!